src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/tests/consolidate_tuning/JvmFlipFlopITest.java

```diff
@@ -0,0 +1,334 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.integTests.tests.consolidate_tuning;
```

Comment:
can we add license header?

Reply:
will add the header in my next PR. Thanks for pointing this out